projects/demo-maps/src/app/route-components/route-example-olperformance/fetchbutton/fetchbutton.component.ts

```diff
@@ -0,0 +1,113 @@
+import { Component, OnInit } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { LayersService, VectorLayer } from '@dlr-eoc/services-layers';
+import { Fill as olFill, Stroke as olStroke, Style as olStyle } from 'ol/style';
+import simplify from '@turf/simplify';
+import bbox from '@turf/bbox';
```

Comment:
can you try to use openLayers for this or some functions from the map-ol.service.ts so we dont have to include another library in the dependencies. 

e.g. [geoJsonToFeatures](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/projects/map-ol/src/lib/map-ol.service.ts#L1263) and then [getFeaturesExtent](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/projects/map-ol/src/lib/map-ol.service.ts#L1192)